dias/analyzers/sensitivity_analyzer.py

```diff
@@ -0,0 +1,273 @@
+from dias import CHIMEAnalyzer
+from datetime import datetime
+import calendar
+from caput import config
+from dias.utils.string_converter import str2timedelta, datetime2str
+from ch_util import data_index
+
+import os
+import sys
+import glob
+import time
+import subprocess
+import gc
+
+import h5py
+import numpy as np
+import scipy.constants
+
+from collections import Counter
+from collections import defaultdict
+
+from ch_util import andata
+from ch_util import tools
+from ch_util import ephemeris
+
+
+class SensitivityAnalyzer(CHIMEAnalyzer):
+    """SensitivityAnalyzer.
+    This subclass of dias.analyzer.Analyzer describes the analyzer.
+    """
+    acq_dir         = config.Property(proptype=str,   default='/mnt/gong/archive')
+    period          = config.Property(proptype=str,  default="24h")
```

Comment:
```suggestion
```
This is done by the base class. Note that there it is of type datetime.timedelta